lib/smart_scheduling_links_test_kit.rb

```diff
@@ -16,9 +16,18 @@ class Suite < Inferno::TestSuite
     )
     version VERSION
 
-    validator do
-      url ENV.fetch('VALIDATOR_URL')
-      exclude_message { |message| message.type == 'info' }
+    VALIDATION_MESSAGE_FILTERS = [
+      /\A\S+: \S+: URL value '.*' does not resolve/
+    ].freeze
+
+    fhir_resource_validator do
+      igs 'igs/package.tgz'
```

Comment:
This is going to get renamed, right?

Reply:
Yeah I went back & forth on that but after realizing the deployment challenges, I'll change this to `smart_scheduling_links_ig.tgz`